pkg/monitortests/network/legacynetworkmonitortests/networking.go

```diff
@@ -92,10 +93,33 @@ func testPodSandboxCreation(events monitorapi.Intervals, clientConfig *rest.Conf
 		failures = append(failures, fmt.Sprintf("error determining platform type: %v", err))
 	}
 
+	// Filter out a list of node NotReady events, we use these to ignore some other potential problems
+	nodeNotReadyIntervals := events.Filter(func(eventInterval monitorapi.Interval) bool {
+		return eventInterval.Source == monitorapi.SourceNodeMonitor &&
+			eventInterval.StructuredLocator.Type == monitorapi.LocatorTypeNode &&
+			eventInterval.StructuredMessage.Reason == monitorapi.NodeNotReadyReason
+	})
+	logrus.Infof("found %d node NotReady intervals", len(nodeNotReadyIntervals))
+
 	for _, event := range events {
+
 		if !strings.Contains(event.Message, "reason/FailedCreatePodSandBox Failed to create pod sandbox") {
 			continue
 		}
+
+		// Skip pod sandbox failures when nodes are updating
+		var foundOverlap bool
+		for _, nui := range nodeNotReadyIntervals {
+			if nui.From.Before(event.From) && nui.To.After(event.To) {
+				logrus.Infof("%s was found to overlap with %s, ignoring pod sandbox error as we expect these if the node is NotReady", event, nui)
```

Comment:
I'm thinking we are good but wondering if we want / need to consider partial overlap where either from and to are the same or potentially nui.to is after event.From but before event.to?  Probably not too likely so I  could go either way.

Reply:
Lets play by ear once this is in the wild, we might want to open it up more once we see some data and have the charts finally.